packages/zipkin-transport-kafka/package.json

```diff
@@ -10,7 +10,7 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/openzipkin/zipkin-js",
   "dependencies": {
-    "kafka-node": "^0.3.2"
+    "kafka-node": "^0.5.9"
```

Comment:
`"^0.5.0"`? Internally we depend on `0.5.3` locked (although that might be ok to bump)